pkg/asset/ignition/node.go

```diff
@@ -91,3 +92,48 @@ func ConvertToAppendix(file *igntypes.File) {
 	file.Contents = igntypes.Resource{}
 	file.Overwrite = ignutil.BoolToPtr(false)
 }
+
+// AppendVarPartition appends a /var partition to the ignition configuration to avoid growfs.
+func AppendVarPartition(config *igntypes.Config) {
+	// See https://docs.fedoraproject.org/en-US/fedora-coreos/storage/#_setting_up_separate_var_mounts
```

Comment:
For OCP here, a more authoritative link is the one the OCP docs link above.

Reply:
Fixed.